src/react-components/room/MicSetupModal.js

```diff
@@ -101,18 +102,18 @@ export function MicSetupModal({
         <ToolbarButton
           icon={soundPlaying ? <VolumeHighIcon width={48} height={48} /> : <VolumeOffIcon width={48} height={48} />}
           label="Click to Test Audio"
-          preset={soundPlaying && "blue"}
+          preset={soundPlaying ? "blue" : "basic"}
           className={styles.largeToolbarButton}
+          onClick={onPlaySound}
           large
         />
       </div>
-      <SelectInputField
-        disabled={!microphoneEnabled}
-        value={selectedMicrophone}
-        options={microphoneOptions}
-        onChange={onChangeMicrophone}
-      />
-      <ToggleInput label="Mute My Microphone" value={microphoneMuted} onChange={onChangeMicrophoneMuted} />
+      {microphoneEnabled && (
+        <>
+          <SelectInputField value={selectedMicrophone} options={microphoneOptions} onChange={onChangeMicrophone} />
+          <ToggleInput label="Mute My Microphone" value={microphoneMuted} onChange={onChangeMicrophoneMuted} />
```

Comment:
A note on styling -- wonder if we can make the enabled state more obvious. The difference between these two is subtle, and if it weren't for the mic icon changing I might not know which is which, especially because of how it interacts with the colorful focus border when you first click it:

![image](https://user-images.githubusercontent.com/4072106/94281944-1d046200-ff04-11ea-914e-0849840db920.png)

![image](https://user-images.githubusercontent.com/4072106/94282014-32798c00-ff04-11ea-9b1a-a5d38a1efe8a.png)

Might just be me -- I get mixed up every time I use the hardware mute toggle on my headphones too:
![image](https://user-images.githubusercontent.com/4072106/94282903-3b1e9200-ff05-11ea-910d-6d9704a3bd3d.png)
^ This is what it shows when it is muted. I have to push it up towards the red in order for it to be unmuted.

Reply:
Yeah, let's get Megan's feedback on this. Toggle switches are hard for this exact reason.